src/main/java/org/isf/medicalinventory/manager/MedicalInventoryManager.java

```diff
@@ -360,6 +366,105 @@ public void validateMedicalInventoryRow(MedicalInventory inventory, List<Medical
 		}
 	}
 
+	/**
+	 * Validate the Inventory rows of inventory ward.
+	 *
+	 * @param inventory the {@link MedicalInventory}
+	 * @param inventoryRowSearchList the list of {@link MedicalInventory}
+	 * @throws OHServiceException
+	 */
+	public void validateMedicalWardInventoryRow(MedicalInventory inventory, List<MedicalInventoryRow> inventoryRowSearchList) throws OHServiceException {
+		LocalDateTime movFrom = inventory.getLastModifiedDate();
```

Comment:
the movFrom should not be the `lastModifiedDate`, because that is just for auditing purpose. It should be the `inventoryDate`